cli/exchange/nmp.go

```diff
@@ -62,23 +62,24 @@ func NMPNew() {
 		`{`,
 		`  "label": "",                               /* ` + msgPrinter.Sprintf("A short description of the policy.") + ` */`,
 		`  "description": "",                         /* ` + msgPrinter.Sprintf("(Optional) A much longer description of the policy.") + ` */`,
+		`  "constraints": [                           /* ` + msgPrinter.Sprintf("(Optional) A list of constraint expressions of the form <property name> <operator> <property value>,") + ` */`,
+		`    "myproperty == myvalue"                  /* ` + msgPrinter.Sprintf("separated by boolean operators AND (&&) or OR (||).") + `*/`,
+		`  ],`,
 		`  "properties": [                            /* ` + msgPrinter.Sprintf("(Optional) A list of policy properties that describe this policy.") + ` */`,
 		`    {`,
 		`      "name": "",`,
 		`      "value": null`,
 		`    }`,
 		`  ],`,
-		`  "constraints": [                           /* ` + msgPrinter.Sprintf("(Optional) A list of constraint expressions of the form <property name> <operator> <property value>,") + ` */`,
-		`    "myproperty == myvalue"                  /* ` + msgPrinter.Sprintf("separated by boolean operators AND (&&) or OR (||).") + `*/`,
-		`  ],`,
 		`  "patterns": [                              /* ` + msgPrinter.Sprintf("(Optional) This policy applies to nodes using one of these patterns.") + ` */`,
 		`    ""`,
 		`  ],`,
 		`  "enabled": false,                          /* ` + msgPrinter.Sprintf("Is this policy enabled or disabled.") + ` */`,
+		`  "start": "<RFC3339 timestamp> | now",      /* ` + msgPrinter.Sprintf("When to start an upgrade, default \"now\".") + ` */`,
+		`  "duration": 0,                             /* ` + msgPrinter.Sprintf("Enable agents to randomize upgrade start time within start + duration, default 0.") + ` */`,
```

Comment:
`duration` has been changed to `startWindow`. Please check here for the latest https://github.com/open-horizon/exchange-api/issues/556.